lib/nanoc/cli/commands/view.rb

```diff
@@ -52,6 +52,10 @@ def run
         run Rack::File.new(site.config[:output_dir])
       end.to_app
 
+      # Print a link
+      url = "http://#{options[:host] || 'localhost'}:#{options_for_rack[:Port]}/"
```

Comment:
Oh huh I had no idea http://0.0.0.0:3000/ would actually work in a browser, turns out it does. Feel free to change to your preferred form.